pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
```

Comment:
shall we add a tests for German language now?

Reply:
Most of the tests have already been extended for German (and the other requested languages), see https://github.com/WSE-research/LinguaF/pull/19/files#diff-ad414dbb73de3f5749cf12f9d21f036319bebbf00443a9c2c23a67fb6edd8ae9R98 for example (you have to load the diff manually). Are you referring to these types of tests, or are you thinking of different ones?

Reply:
All good, thanks!